quinn-proto/src/stream.rs

```diff
@@ -91,12 +91,11 @@ impl Send {
             }
             | SendState::ResetRecvd {
                 ref mut stop_reason,
-            } => match stop_reason.take() {
-                Some(error_code) => {
+            } => {
+                if let Some(error_code) = stop_reason.take() {
```

Comment:
Does `cargo fmt` mandate the surrounding braces here? It seems like it could do without them.

Reply:
Yep, I was surprised too, but rustfmt insisted extra curly braces are required. Go figure.